build-scripts/bundle.cjs

```diff
@@ -215,7 +220,13 @@ module.exports.config = {
     return {
       name: "frontend" + nameSuffix(latestBuild),
       entry: {
-        service_worker: "./src/entrypoints/service_worker.ts",
+        "service-worker":
+          !env.useRollup() && !latestBuild
+            ? {
+                import: "./src/entrypoints/service-worker.ts",
+                layer: "sw",
```

Comment:
How do these layers work, can't find anything about it in the webpack docs...

Reply:
Yeah the docs on it are quite sparse.  It's more or less a tag/label for a module.  You can assign it by entry like this, or [via a condition in the module config](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkslayer).  Then you can act on it [in cache groups](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkslayer), by issuer layer in the module config (like I do here to change the loader options), and they also get grouped in the stats output.